### PR TITLE
Add Fullscreen for apple and android

### PR DESCRIPTION
**Pull request by Joe2824**
add apple and android fullscreen for web app

### CODE REVIEW

**Comment by Felli** [comment]
You you explain why this is needed?

**Comment by Joe2824** [comment]
With these lines HumHub use the standalone mode to look more like a native app. For iOS Safari there is no browser URL Textfield at the top of the screen or button bar. Only the status bar appears in black on the screen.
The mobile-web-app-capable is for Android and have the same use.

img_0109
img_0108

**Comment by Felli** [comment]
Thanks for giving your opinion, as I'm not in control of any of HumHub's repositories, I'll look into this and talk with the Development team about adding this. 👍

File: themes/HumHub/views/layouts/head.php [hunk 1]
```diff
@@ -20,3 +20,13 @@
 <meta name="msapplication-TileImage" content="<?= $this->theme->getBaseUrl(); ?>/ico/ms-icon-144x144.png">
 <meta name="theme-color" content="#ffffff">
 <!-- end: Favicon and Touch Icons -->
+
+<!-- start: Apple Fullscreen and Webapp Title -->
+<meta name="apple-mobile-web-app-title" content="HumHub" />
```
Comment (buddh4) [inline themes/HumHub/views/layouts/head.php]: Should rather contain `Html::encode(Yii::$app->name)` instead of `humhub`

**Comment by buddh4** [comment]
Thanks https://github.com/humhub/humhub/commit/d40f1b12efca8188d256c75061dcd06acab08338

**Pull request closed without merging**